### ISSUE TITLE
ACCUMULO_JAVA_OPTS, does it work?

### ISSUE THREAD

**Issue by dlmarion** [apache/accumulo#2383]
**Describe the bug**
I'm setting the following bash environment variable:

```
export ACCUMULO_JAVA_OPTS=("-Dotel.traces.exporter=jaeger" "-javaagent:/path/to/opentelemetry-javaagent-1.7.1.jar")
```
[This](https://github.com/apache/accumulo/blob/main/assemble/conf/accumulo-env.sh#L70) says to do that. However, I don't see those arguments in my process. Furthermore, in the `BUGS` section in the `bash` man page, it says:

```
Array variables may not (yet) be exported.
```

How do I set this environment variable for it to work?

**Versions (OS, Maven, Java, and others, as appropriate):**
 - Affected version(s) of this project: 2.1.0-SNAPSHOT
 - OS: CentOS7

**Comment by ctubbsii** [comment]
Yeah, that's a bug. The `accumulo-env.sh` file is supposed to be sourced, which does pass the variables correctly to the script that sourced it, but exporting does not. Any script that tries to work that way should be fixed. The `accumulo-env.sh` script shouldn't be exporting `JAVA_OPTS` or relying on `ACCUMULO_JAVA_OPTS` to be an array.

**Comment by dlmarion** [comment]
Closed by #2391
